mission/para_server_init.sqf

```diff
@@ -109,8 +109,8 @@ vn_site_objects = [];
 
 // Set desired number of simultaneously active zones.
 vn_mf_targetNumberOfActiveZones = 1;
-// Set number of enemies per player.
-para_g_enemiesPerPlayer = 2;
+// Set number of enemies per player. Scale the default value by the percentage set in the config options.
+para_g_enemiesPerPlayer = ((["ai_scaling", 100] call BIS_fnc_getParamValue) / 100) * 2;
```

Comment:
We'll need to look into what to set this as.

Reply:
@wescottj98 I don't think there's a way to test this against peaks hours server population without putting it live, seeing what happens and tweaking unfortunately.